acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ThreadBesuNodeRunner.java

```diff
@@ -396,4 +321,229 @@ public void startConsoleCapture() {
   public String getConsoleContents() {
     throw new RuntimeException("Console contents can only be captured in process execution");
   }
+
+  @Module
+  @SuppressWarnings("CloseableProvides")
+  static class BesuControllerModule {
```

Comment:
Wondering about this being nested vs pulled out into its own class...keen to hear your thoughts.

Are we likely to end up with lots of Modules attached to classes or just a few at the top level?